guidelines/index.html

```diff
@@ -78,1254 +106,1196 @@ <h3>Section status levels</h3>
-					<h4>Avoid manipulation</h4>
-					<p class="guideline-text">Tasks can be completed without navigating misinformation or redirections.</p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Go back in process</h4>
-					<p class="guideline-text">The interface supports stepping backwards in a process and returning to the current point without data loss.</p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Inform at start</h4>
-					<p class="guideline-text">Information needed to complete a multi-step process is provided at the start of the process, including:
-					<ul><li>estimated time or number of steps it might take,</li>
-					<li>details of any resources needed to perform the task, and</li>
-					<li>overview of the process and next step.</li></ul>
-					</p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>No cognitive tests</h4>
-					<p class="guideline-text">Tasks, including login/authentication, can be completed without puzzles, calculations, or other cognitive tests (essential exceptions would apply). </p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>No memorization</h4>
-					<p class="guideline-text">Tasks can be completed without memorizing and recalling information from previous stages of the process. </p>
-					<p class="ednote">Needs <a href="#additional_research">additional research</a></p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Optimized processes</h4>
-					<p class="guideline-text">Tasks can be completed without reading or understanding unnecessary content.</p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Optional information</h4>
-					<p class="guideline-text">Tasks can be completed without entering unnecessary information.</p>
-				</section>
-				<section class="guideline"  data-status="exploratory">
-					<h4>Preselections</h4>
-					<p class="guideline-text">Preselections are visible during task completion.</p>
+				<h3>Process and task completion</h3>
+				<section class="guideline">
+					<h4>Avoid cognitive tasks</h4>
+					<p class="guideline-text">Users can complete tasks without needing to memorize nor complete advanced cognitive tasks.</p>
+					<section class="provision" data-status="exploratory">
+						<h5>Allow automated entry</h5>
+						<p class="provision-text">Automated input from user agents, 3rd party tools, or copy-and-paste is not prevented.</p>
+					</section>
+					<section class="provision" data-status="exploratory">
+						<h5>No cognitive tests</h5>
+						<p class="provision-text">Processes, including login/authentication, can be completed without puzzles, calculations, or other cognitive tests (essential exceptions would apply).</p>
```

Comment:
I feel this would be more clear if explicitly 'and': 

```suggestion
						<p class="provision-text">Processes, including login and authentication, can be completed without puzzles, calculations, or other cognitive tests (essential exceptions would apply).</p>
```
or if we used just authentication:

```suggestion
						<p class="provision-text">Processes, including authentication, can be completed without puzzles, calculations, or other cognitive tests (essential exceptions would apply).</p>
```